controllers/components/kafka_topic.go

```diff
@@ -42,7 +46,48 @@ func (kt *KafkaTopic) Delete() (err error) {
 }
 
 func (kt *KafkaTopic) CheckDeviation() (problem, err error) {
-	return //TODO
+	found, err := kt.Exists()
+	if err != nil {
+		return nil, errors.Wrap(err, 0)
+	}
+	if !found {
+		return fmt.Errorf("KafkaTopic named, %s, does not exist.", kt.name), nil
+	}
+
+	// leave this commented line for testing.
+	// topicIn, err := kt.KafkaTopics.GetTopic("platform.inventory.events")
+	topicIn, err := kt.KafkaTopics.GetTopic(kt.name)
```

Comment:
the test failures are caused by this `kt.name`. It should be `kt.Name()` to use the versioned name.

Reply:
Fixed!